src/Sylius/Behat/Context/Setup/ProductAttributeContext.php

```diff
@@ -70,11 +70,16 @@ public function theStoreHasATextProductAttribute($type, $name)
     /**
      * @param string $type
      * @param string $name
-     * @param string $code
+     * @param string|null $code
      */
-    private function createProductAttribute($type, $name, $code = 'PA112')
+    private function createProductAttribute($type, $name, $code = null)
     {
         $productAttribute = $this->productAttributeFactory->createTyped($type);
+
+        if (null === $code) {
+            $code = str_replace(' ', '_', strtoupper($name));
```

Comment:
Good idea 👍  Maybe I could add this in separate PR (and therefore use it in every place its necessary)?